modules/admin_manual/pages/configuration/files/external_storage/ftp.adoc

```diff
@@ -18,14 +18,16 @@ To connect to an FTP server, you will need:
 Your new mountpoint is available to all users by default, and you may restrict access by entering specific users or groups in the *Available for* field.
 
 Optionally, ownCloud can use FTPS (FTP over SSL) by checking `Secure ftps://`. 
-This requires additional configuration with your root certificate, if the FTP server uses https://en.wikipedia.org/wiki/Self-signed_certificate[a self-signed certificate].
+This requires additional configuration with your root certificate, if the FTP server uses
+https://en.wikipedia.org/wiki/Self-signed_certificate[a self-signed certificate].
 See xref:configuration/server/import_ssl_cert.adoc[Importing System-wide and Personal SSL Certificates] for more information.
 
 image:configuration/files/external_storage/ftp.png[ownCloud GUI FTP configuration.]
 
 The external storage `FTP/FTPS` needs the `allow_url_fopen` PHP setting to be set to `1`.
 When having connection problems make sure that it is not set to `0` in your `php.ini`.
-See https://doc.owncloud.com/server/latest/admin_manual/issues/general_troubleshooting.html#label-phpinfo[PHP Version and Information] to learn how to find the right `php.ini` file to edit.
+See xref:configuration/general_topics/general_troubleshooting.adoc#php-version-and-information[PHP Version and Information]
```

Comment:
Is there a specific need to split sentences over multiple lines?

Reply:
see above